test/compress/functions.js

```diff
@@ -0,0 +1,6 @@
+non_ascii_function_identifier_name: {
+    input: {
+        function fooλ(δλ) {}
```

Comment:
Done (deleted comment was just something that was caused by looking wrongly at the test)